### DISCUSSION TITLE
Robocopy could not be found

### DISCUSSION THREAD

**Discussion by kymerea** [ScoopInstaller/Scoop#5518] [Help and Questions] [answered]
#### Current Behavior
When trying to install scoop the following error occurs:
```
Initializing...
Scoop requires 'C:\Windows\System32\Robocopy.exe' to work. Please make sure 'C:\Windows\System32' is in your PATH.
Abort.
```
#### Expected Behavior
To actually install scoop

#### Additional context/output
I have looked at some other bug reports on the repo, I have confirmed that C:\Windows\System32 is in path aswell as Robocopy being present and usable.
Output of `$env:PATH.Split(';')`
```
c:\program files (x86)\common files\oracle\java\javapath
c:\windows\system32
c:\windows
c:\windows\system32\wbem
c:\windows\system32\windowspowershell\v1.0\
c:\windows\system32\openssh\
c:\program files\nvidia corporation\nvidia nvdlisr
c:\program files\qemu
c:\program files\dotnet\
c:\windows\apktool
C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common
C:\Program Files\nodejs\
C:\Program Files\Git\cmd
C:\Windows\System32
C:\Users\hudso\AppData\Local\Microsoft\WindowsApps
C:\Users\hudso\AppData\Local\spicetify
C:\Program Files\platform-tools_r33.0.3-windows\platform-tools
%PyCharm Community Edition%
C:\Users\hudso\AppData\Roaming\npm
C:\Users\hudso\.dotnet\tools
C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1.2\bin
C:\Users\hudso\AppData\Local\Programs\Microsoft VS Code\bin
C:\Windows\System32
```

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:**
Major  Minor  Build  Revision
5      1      22621  963

Edit: 
Robocopy works on CMD but not PS
I have been having some weird issues with PS lately so that may be why this is happening

**Comment by HUMORCE** [comment]
```
where.exe robocopy
```

**Comment by kymerea** [comment]
On CMD is shows: 
`c:\Windows\System32\Robocopy.exe`
Whilst on PS:
A CMD window briefly opens then closes, nothing is outputted in the PowerShell terminal

**Comment by HUMORCE** [comment]
Your environment variable settings seem to be fine, except that `%PyCharm Community Edition%` is undefined.

Try pwsh core: https://github.com/PowerShell/PowerShell#get-powershell

If it cannot be reproduced in pwsh core, then you should fix configs of powershell 5.x

**Reply by rashil2000** [reply]
> Your environment variable settings seem to be fine, except that `%PyCharm Community Edition%` is undefined.

As far as I know, environment variables cannot contain spaces. If you remove the spaces from this variable in the PATH, things should work fine.

**Comment by kymerea** [comment, marked as answer]
Well, after finding a couple other things wrong with my PC, i decided to reinstall Windows which fixed all bugs I had
